content/en/ddsql_reference/ddsql_preview.md

```diff
@@ -0,0 +1,31 @@
+---
+title: DDSQL (Preview)
+code_lang: ddsql_preview
+type: multi-code-lang
+code_lang_weight: 2
+further_reading:
+- link: "/ddsql_editor/"
+  tag: "Documentation"
+  text: "Learn more about the DDSQL Editor"
+---
+
+{{< callout url="https://datadoghq.com/private-beta/ddsql-editor">}}
+DDSQL Editor is in Preview.
+{{< /callout >}}
```

Comment:
I noticed there's no disclaimer on this page about there being two variants of DDSQL - is that intentional?

Reply:
Yes, the product team requested that it be removed